Examples/Algorithms/TrackFinding/src/TrackParamsEstimationAlgorithm.cpp

```diff
@@ -112,13 +117,22 @@ ActsExamples::TrackParamsEstimationAlgorithm::createSeeds(
                                      spacePointsOnTrack[im]->r());
           if (bmDeltaR >= m_cfg.deltaRMin and bmDeltaR <= m_cfg.deltaRMax and
               mtDeltaR >= m_cfg.deltaRMin and mtDeltaR <= m_cfg.deltaRMax) {
-            seeds.emplace_back(*spacePointsOnTrack[ib], *spacePointsOnTrack[im],
-                               *spacePointsOnTrack[it],
-                               spacePointsOnTrack[im]->z());
+            if ((bmDeltaR + mtDeltaR) > maxDeltaR) {
+              maxDeltaR = bmDeltaR + mtDeltaR;
+              bestSPIndices = {ib, im, it};
+              seedFound = true;
+            }
           }
         }
       }
     }
+
+    if (seedFound) {
+      seeds.emplace_back(*spacePointsOnTrack[bestSPIndices[0]],
+                         *spacePointsOnTrack[bestSPIndices[1]],
+                         *spacePointsOnTrack[bestSPIndices[2]],
+                         spacePointsOnTrack[bestSPIndices[1]]->z());
+    }
```

Comment:
Ok, I see, in ATLAS we have some sort of prescription how to build seeds, called PPP, PPS, SSS for Pixel and strip combinations, we might need something like this in the future.

Reply:
Yes, we should have this.